### DISCUSSION TITLE
Translations for [Unknown Album]

### DISCUSSION THREAD

**Discussion by Suoslex** [navidrome/navidrome#3208] [Ideas]
The whole project is well translated, but It would be great if we could have translations for this label as well.
Also, it would be great If we could change this label to our taste, because these square brackets looks kind of scary for a common user.

**Comment by deluan** [comment]
> The whole project is well translated, but It would be great if we could have translations for this label as well.
> Also, it would be great If we could change this label to our taste, because these square brackets looks kind of scary for a common user.

They are meant to be scary :P 

The reason these labels are currently not translatable is that they are set server side, and all translations are only on the frontend/UI.

I may change this in a future release.
